public/static/locales/es/common.json

```diff
@@ -1,37 +1,39 @@
 {
```

Comment:
Please don't make any changes to this file. This file will be automatically updated from crowdin based on the changes you make on `public/static/locales/en/common.json`